controller/raft/mesh/mesh.go

```diff
@@ -247,7 +247,7 @@ type impl struct {
 	closeNotify          chan struct{}
 	closed               atomic.Bool
 	raftAccepts          chan net.Conn
-	bindHandler          channel.BindHandler
+	bindHandler          concurrenz.AtomicValue[channel.BindHandler]
```

Comment:
are these changes expected? they are more than "devs and changelog"

Reply:
It's a 'just in case' thread safety fix. Didn't turn out to be the issue, but it's still the safe thing to do, so I left it in